docs/uml-class-diagrams/database/production/DatabaseCommInterfaceImplementation/DatabaseCommInterfaceImplementation.puml

```diff
@@ -2,10 +2,21 @@
 
 package edu.oakland.production.database {
     class DatabaseCommInterfaceImplementation {
+        - DatabaseCommManager databaseCommManager
+    	- int rfidData
+    	- LocationDataPoint locDataPoint
+    	- TrackData trackData
+    	- String mode
```

Comment:
Does this class need to know all of the private variables? If not, then remove them.
```suggestion
      - DatabaseCommManager databaseCommManager
```